docs/skopeo-standalone-sign.1.md

```diff
@@ -26,7 +26,7 @@ $
 ```
 
 ## SEE ALSO
-skopeo(1), skopeo-copy(1)
+skopeo(1), skopeo-copy(1), containers\-signature(5)
```

Comment:
OK, this one has a slash, but `containers-auth.json` does not.  I can't remember for sure, but I don't think you need the backslashes in this case.

Reply:
I removed the `\` from all of them, since it was inconsistent, and I don't think we need them there.